new resource `azurerm_monitor_action_rule_action_group`, `azurerm_monitor_action_rule_suppression` (#6563)
terraform-providers/terraform-provider-azurerm: website/docs/r/monitor_action_rule.html.markdown

  name                = "example-amar"
  resource_group_name = azurerm_resource_group.example.name
  type                = "ActionGroup"
  action_group_id     = azurerm_monitor_action_group.example.id

  scope {
    type         = "ResourceGroup"
    resource_ids = [azurerm_resource_group.example.id]
  }

  tags = {
    foo = "bar"
  }
}
```

## Argument Reference

The following arguments are supported:

* `name` - (Required) Specifies the name of the Monitor Action Rule. Changing this forces a new resource to be created.

* `resource_group_name` - (Required) Specifies the name of the resource group in which the Monitor Action Rule should exist. Changing this forces a new resource to be created.

* `type` - (Required) Specifies the type of the Action Rule. Possible values are `Suppression`, `ActionGroup` and `Diagnostics`. Changing this forces a new resource to be created.

* `description` - (Optional) Specifies a description for the Action Rule.

* `enabled` - (Optional) Is the Action Rule enabled? Defaults to `true`.

* `action_group_id` - (Optional) Specifies the resource id of monitor action group. Required only if `type` is `ActionGroup`.

> * `suppression` - (Optional) One `suppression` as defined below. Required only if `type` is `Suppression`.

Reviewer: I think for consistency we should word this as 
```suggestion
* `suppression` - (Optional) A `suppression` block as defined below. Required only if `type` is `Suppression`.
```

* `end_date` - (Required) specifies the UTC end date of recurrence range. The format should be `MM/DD/YYYY`.

* `end_time` - (Required) specifies the UTC end time of recurrence range. The format should be `HH:MM:SS`.

* `recurrence_weekly` - (Optional) specifies the list of dayOfWeek to recurrence. Possible values are `Sunday`, `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday` and  `Saturday`.

* `recurrence_monthly` - (Optional) specifies the list of dayOfMonth to recurrence. Possible values are between `1` - `31`. Required if `recurrence_type` is `Monthly`.

---

The `alert_context` block supports the following:

* `operator` - (Required) operator for a given condition. Possible values are `Equals`, `NotEquals`, `Contains` and `DoesNotContain`.

* `values` - (Required) A list of values to match for a given condition.

---

The `alert_rule_id` block supports the following:

* `operator` - (Required) operator for a given condition. Possible values are `Equals`, `NotEquals`, `Contains` and `DoesNotContain`.

* `values` - (Required) A list of values to match for a given condition.

---

The `description` block supports the following:

> * `operator` - (Required) operator for a given condition. Possible values are `Equals`, `NotEquals`, `Contains` and `DoesNotContain`.

Reviewer: ```suggestion
* `operator` - (Required) The operator for a given condition. Possible values are `Equals`, `NotEquals`, `Contains`, and `DoesNotContain`.
```

* `values` - (Required) A list of values to match for a given condition.

---

The `monitor` block supports the following:

* `operator` - (Required) operator for a given condition. Possible values are `Equals`, `NotEquals`.

* `values` - (Required) A list of values to match for a given condition. Possible values are `Fired`, `Resolved`.

---

The `monitor_service` block supports the following:

* `operator` - (Required) operator for a given condition. Possible values are `Equals`, `NotEquals`.

* `values` - (Required) A list of values to match for a given condition. Possible values are `ActivityLog Administrative`, `ActivityLog Autoscale`, `ActivityLog Policy`, `ActivityLog Recommendation`, `ActivityLog Security`, `Application Insights`, `Azure Backup`, `Data Box Edge`, `Data Box Gateway`, `Health Platform`, `Log Analytics`, `Platform` and `Resource Health`.

---

The `severity` block supports the following:

> * `operator` - (Required) operator for a given condition. Possible values are `Equals`, `NotEquals`.

Reviewer: Grammar and we should capitalize first letter
```suggestion
* `operator` - (Required) The operator for a given condition. Possible values are `Equals` and `NotEquals`.
```

> * `values` - (Required) A list of values to match for a given condition. Possible values are `Sev0`, `Sev1`, `Sev2`, `Sev3` and `Sev4`.

Reviewer: We should use an oxford comma here
```suggestion
* `values` - (Required) A list of values to match for a given condition. Possible values are `Sev0`, `Sev1`, `Sev2`, `Sev3`, and `Sev4`.
```
